pkgs/top-level/python-new-packages.nix

```diff
@@ -0,0 +1,62 @@
+{ pkgs, stdenv }:
+
+with pkgs;
+
+let
+
+  interpreters = rec {
+    cpython26 = callPackage ../development/interpreters/python/cpython/2.6 {
+      inherit (darwin) CF configd;
+      self = cpython26;
+    };
+    cpython27 = callPackage ../development/interpreters/python/cpython/2.7 {
+      inherit (darwin) CF configd;
+      self = cpython27;
+    };
+    cpython33 = callPackage ../development/interpreters/python/cpython/3.3 {
+      inherit (darwin) CF configd;
+      self = cpython33;
+    };
+    cpython34 = callPackage ../development/interpreters/python/cpython/3.4 {
+      inherit (darwin) CF configd;
+      self = cpython34;
+    };
+    cpython35 = callPackage ../development/interpreters/python/cpython/3.5 {
+      inherit (darwin) CF configd;
+      self = cpython35;
+    };
+    cpython36 = callPackage ../development/interpreters/python/cpython/3.6 {
+      inherit (darwin) CF configd;
+      self = cpython36;
+    };
+    pypy27 = callPackage ../development/interpreters/python/pypy/2.7 {
+      self = pypy27;
+    };
+  };
+
+in rec {
+# FIXME: use map
+
+  cpython26 = callPackage ../development/python-modules {
+    interpreter = interpreters.cpython26;
+  };
+  cpython27 = callPackage ../development/python-modules {
+    interpreter = interpreters.cpython27;
+  };
+  cpython33 = callPackage ../development/python-modules {
+    interpreter = interpreters.cpython33;
+  };
+  cpython34 = callPackage ../development/python-modules {
+    interpreter = interpreters.cpython34;
+  };
+  cpython35 = callPackage ../development/python-modules {
+    interpreter = interpreters.cpython35;
+  };
+  cpython36 = callPackage ../development/python-modules {
+    interpreter = interpreters.cpython36;
+  };
+  pypy27 = callPackage ../development/python-modules {
+    interpreter = interpreters.pypy27;
+  };
+
+}
```

Comment:
This is just `lib.mapAtttrs (_: interpreter: callPackage ../development/python-modules { inherit inherpreter; })` afaik